src/consensus/ethash/difficulty.rs

```diff
@@ -0,0 +1,118 @@
+use crate::models::*;
+use ethereum_types::*;
+use std::cmp::max;
+
+const MIN_DIFFICULTY: u64 = 131_072;
+
+pub struct BlockDifficultyBombData {
+    pub delay_to: BlockNumber,
+}
+
+#[allow(clippy::too_many_arguments)]
```

Comment:
in theory, what would be a way to resolve this warning?

Reply:
Grouping the last three parameters into a struct. It's possible, I'm just not sure how justified it is. If only to make clippy shut up - it's much easier to ignore the warning.